codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttProperties.java

```diff
@@ -67,6 +67,14 @@
         CORRELATION_DATA(0x09),
         AUTHENTICATION_DATA(0x16);
 
+        private static final MqttPropertyType[] VALUES = new MqttPropertyType[43];
```

Comment:
I prefer to move the array allocations to the static block, so you don't have to hard code the array length like this. Same in `MqttQoS`.

Reply:
:+1: